src/blocks/tipset.rs

```diff
@@ -90,6 +90,17 @@ impl TipsetKey {
             .collect::<Vec<_>>()
             .join(", ")
     }
+
+    /// Formats tipset key to match the Lotus display.
+    pub fn to_lotus(&self) -> String {
```

Comment:
```suggestion
    pub fn format_lotus(&self) -> String {
```
nit: I'd say it's more expressive this way. `to_lotus` suggests some other type.